index.html

```diff
@@ -116,6 +116,28 @@ <h2>A project for first-time contributions</h2>
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
         <!-- ________ matheustelino card START ________  -->
+        <div class="card">
+          <p class="name">Muhmmad Abrar</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/iMuhmmadAbrar" target="_blank">Your handle</a>
+          </p>
+          <p class="about">Hey, I am an undergrad. Looking forward to develop web applications and focused on deep learning. Hope these links help you.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="Stack Overflow">Doubt clearances made simple.</a>
+              </li>
+              <li>
+                <a href="https://roadmap.sh/" target="_blank" title="Developer Roadmap">Roadmap for skills</a>
+              </li>
+              <li>
+                <a href="https://www.freecodecamp.org/" target="_blank" title="FreeCodeCamp">Learning materials for free.</a>
+              </li>
+            </ul>
+          </div>
+        </div>

```

Comment:
Hi, add a comment showing you name and the phrase "card end". Just like you did for card start